### ISSUE TITLE
instance_eval error in reminder-text.json.rb

### ISSUE THREAD

**Issue by royfielding** [apache/whimsy#96]
Sadly, my guess in #95 wasn't sufficient. I am still getting an error on send final reminders.

The error log says

```
App 13507 output: _ERROR #<SecurityError: Insecure operation - instance_eval>
App 13507 output: _WARN /x1/srv/whimsy/www/board/agenda/views/actions/reminder-text.json.rb:28:in `_evaluate'
App 13507 output: _WARN /x1/srv/whimsy/www/board/agenda/routes.rb:465:in `block in <top (required)>' 
```

**Comment by royfielding** [comment]
Oh, hold on, it might be because I removed "reminder" from the subject.

**Comment by royfielding** [comment]
Fixed by 3a0aa52bebbcca75508ef983711df550db6483b3
